epf/FileProcessor.cpp

```diff
@@ -70,7 +68,7 @@ void FileProcessor::run()
             VoxelKey cellIndex = m_grid.key(p.x(), p.y(), p.z());
             if (cellIndex != cell->key())
             {
-                cell = m_cellMgr.get(cellIndex);
+                cell = m_cellMgr.get(cellIndex, cell);
```

Comment:
We are passing in the cell reference here because it might change, correct? A note about why that could happen would be useful.

Reply:
No, that's not what's happening. The cell pointer is const. The cell that's the argument indicates a pointer to the cell whose data we don't want flushed. If we don't keep that cell's data around, the next line that calls `copyPoint` can copy bad data, which was the original bug.

I'll add a comment.